README.md

```diff
@@ -10,6 +10,9 @@ comments.  For more information about OpenShift, visit https://openshift.redhat.
 or the OpenShift support page
 https://openshift.redhat.com/support.
 
+RHC is typically built on Ruby 1.8.7.  If you are running into any dependency 
```

Comment:
Good point.  I'll replace that with something mentioning the rubygems.org rhc is built on 1.8.7, but the gem can be built on 1.8, 1.9, and 2.0.